### PR TITLE
ELEMENTS-1289: introduce importHTML

### PR DESCRIPTION
**Pull request by nelsonsilva**
Helper method to import HTML to be used when importing Web UI bundles.

### CODE REVIEW

File: ui/import-href.js [hunk 1]
```diff
@@ -93,3 +93,25 @@ export const importHref = function(href, onload, onerror, optAsync) {
...
+ * @param {string} html HTML to import.
+ */
+export const importHTML = (html) => {
+  const tmpl = document.createElement('template');
+  tmpl.innerHTML = html;
+  [...tmpl.content.children].forEach((el) => {
+    if (el.tagName === 'SCRIPT' && !el.src) {
+      const script = document.createElement('script');
+      [...el.attributes].forEach((attr) => script.setAttribute(attr.name, attr.value));
+      script.setAttribute('src', `data:text/javascript;charset=utf-8,${encodeURIComponent(el.textContent)}`);
```
Comment (Gabez0r) [inline ui/import-href.js:112]: Sorry, didn't have time to test it, but: does this make it hard to debug the script  on the borwser?

**Review by Gabez0r — COMMENTED** [review]
LGTM, but were would we be using it? Should we consider using it for loading `nuxeo-web-ui-bundle.html`?

**Review by richardsd — DISMISSED** [review]

File: ui/import-href.js (hunk 1 above)
Comment (nelsonsilva) [reply ui/import-href.js:112]: without adding some source maps I guess it does, yes but I think the logics in the importHref polyfill do that so we can improve this too.

**Comment by nelsonsilva** [comment]
> LGTM, but were would we be using it? Should we consider using it for loading `nuxeo-web-ui-bundle.html`?

Yes, we will use it for loading all bundles that are currently being loaded by using directly imported HTML, like the Web UI bundle and Aspera and Retention addons iirc

File: ui/import-href.js [hunk 2]
```diff
@@ -93,3 +93,25 @@ export const importHref = function(href, onload, onerror, optAsync) {
   }
   return link;
 };
+
+/**
+ * Convenience method for importing inline HTML.
+ *
+ * This method imports all the elements to the document's body
```
Comment (Gabez0r) [inline ui/import-href.js:100]:
```suggestion
 * This method imports all the elements to the document's head
```

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**